infra/serializers.py

```diff
@@ -0,0 +1,30 @@
+'''Define how to serialize our models.'''
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+import infra.models
+
+
+User = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
Why are you re-defining the UserSerializer?
If you need it, just import it from auth module.